tests/test_structured_output.py

```
@@ -0,0 +1,205 @@
+"""Tests for structured output using Pydantic BaseModel"""
+
+import pytest
+from typing import Optional, List
+from pydantic import BaseModel, Field
+
+from pyhub.llm import LLM
```

Ruff (0.11.9)

`pyhub.llm.LLM` imported but unused

Remove unused import: `pyhub.llm.LLM`

(F401)